mix.exs

```diff
@@ -34,14 +34,14 @@ defmodule Warehouse.MixProject do
   defp deps do
     [
       {:amqp, "~> 2.0", override: true},
-      {:appsignal, "~> 1.0"},
-      {:bottle, github: "system76/bottle", ref: "b3b78b6"},
+      {:bottle, github: "system76/bottle", ref: "0fc599f"},
       {:broadway_rabbitmq, "~> 0.6.0"},
       {:decimal, "~> 1.9.0", override: true},
       {:ecto_enum, "~> 1.4"},
       {:ecto_sql, "~> 3.5"},
       {:ex_aws_sqs, "~> 3.2"},
       {:ex_aws, "~> 2.1.6"},
+      {:flow, "~> 1.1"},
```

Comment:
Oh my. :eyes:

Reply:
I'm not going to use this. It was something I was trying when we were running into those issues before :grin:

Reply:
Ah alright. Wasn't sure if you were going to do something fancy